examples/h_mountain.py

```diff
@@ -0,0 +1,170 @@
+from gusto import *
+from firedrake import FunctionSpace, as_vector, \
+    VectorFunctionSpace, PeriodicIntervalMesh, ExtrudedMesh, \
+    SpatialCoordinate, exp, pi, cos, Function, conditional, Mesh, sin, op2, sqrt
+import sys
+
+dt = 5.0
+if '--running-tests' in sys.argv:
+    tmax = dt
+else:
+    tmax = 15000.
+
+res = 10
+nlayers = res*20  # horizontal layers
+columns = res*12  # number of columns
+L = 240000.
+m = PeriodicIntervalMesh(columns, L)
+
+# build volume mesh
+H = 50000.  # Height position of the model top
+ext_mesh = ExtrudedMesh(m, layers=nlayers, layer_height=H/nlayers)
+Vc = VectorFunctionSpace(ext_mesh, "DG", 2)
+coord = SpatialCoordinate(ext_mesh)
+x = Function(Vc).interpolate(as_vector([coord[0], coord[1]]))
```

Comment:
Do you still need lines 23 and 24 since below you have 
`x, z = SpatialCoordinate(ext_mesh)`?

Reply:
Hmm you are probably right. I didn't really have a proper look at this part since I simply copied it from nh_mountain.